src/v8.ts

```diff
@@ -365,6 +365,15 @@ export const Todoist = (token: string, userOptions = defaultOptions) => {
     get: (options: any) => request({ url: `${options.endpoint}/activity/get`, query: options }),
   }
 
+  const completedItems = {
+    get: async (opts: any) => {
```

Comment:
Why `get()` rather than `getAll()`? The later has the advantage of conserving a 1-1 mapping with the API's method names.

Reply:
Idem for `completedItems` vs `completed`.